asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -57,14 +63,37 @@ def get_enhanced_mcs():
         }
     )
 
-    return mcs
+    return df
+
+
+def get_total_cumsums(data, installation_date_col):
+    """
+    Gets cumulative MCS installations for Wales.
+
+    Returns:
+        pd.Dataframe containing cumulative MCS installations for Wales over time.
+
+    """
+    data["n"] = 1
+    cumulative_total = data.groupby(installation_date_col)["n"].sum().reset_index()
+
+    # Sort by date ascending
+    cumulative_total = cumulative_total.sort_values(installation_date_col)
 
+    # Get cumulative total
+    cumulative_total["cumsum"] = cumulative_total.n.cumsum()
+    cumulative_total = cumulative_total.loc[
+        cumulative_total[installation_date_col] >= "2015-01-01"
+    ].reset_index(drop=True)
+    cumulative_total = cumulative_total.rename(columns={installation_date_col: "date"})
+    cumulative_total["colour"] = 1  # add single colour category for plotting
 
-# load enhanced MCS as part of this script, so only needs to be done once
-enhanced_mcs = get_enhanced_mcs()
+    return cumulative_total
 
 
-def cumsums_by_variable(variable, new_var_name, data=enhanced_mcs):
+def cumsums_by_variable(
+    variable, new_var_name, data, installation_date_col="HP_INSTALL_DATE"
```

Comment:
Any reason why you chose to have the HP_INSTALL_DATE as default instead of commision_date?

Reply:
No reason, can be either.